components/tidb_query/src/rpn_expr/impl_op.rs

```diff
@@ -343,4 +352,26 @@ mod tests {
             assert_eq!(output, expect_output, "{:?}, {:?}", arg, sig);
         }
     }
+
+    #[test]
+    fn test_left_shift() {
+        let test_cases = vec![
```

Comment:
Suggest to add these test cases: 

- `None, None => None`
- `Some(123), Some(2) => Some(492)` 
- `Some(-123), Some(-1) => Some(0)`